ansible/playbooks/roles/upgrade/tasks/kubernetes/verify-upgrade.yml

```diff
@@ -64,3 +64,5 @@
   check_mode: true
   register: cgroup_driver
   failed_when: cgroup_driver.changed
+  when:
```

Comment:
Isn't the whole verify-upgrade part executed on the last iteration?

Reply:
I see it's on every iteration at the end: 
https://github.com/epiphany-platform/epiphany/blob/3d86a042725dfd63ee26ea0eb556057c79beae3f/ansible/playbooks/roles/upgrade/tasks/kubernetes/upgrade-master0.yml#L100
https://github.com/epiphany-platform/epiphany/blob/3d86a042725dfd63ee26ea0eb556057c79beae3f/ansible/playbooks/roles/upgrade/tasks/kubernetes/upgrade-masterN.yml#L54